kwill307-newyork.md

```diff
@@ -0,0 +1,4 @@
+## Where to Eat
+- Sponge Docks
+- Dunedin Brewery
```

Comment:
[jmick4] What about Southern Tier... can't forget the Creme Brulee Stout. :beer: